### DISCUSSION TITLE
Change font colors where?

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#83] [Q&A]
So what's the best way to change my font colors?
The files in mediacms/static/css are all minimized.

**Comment by mgogoulos** [comment]
We'll add documentation on the docs page we're preparing, on the meanwhile @styiannis you might want to give a quick response into this!

**Reply by Dan1ell** [reply]
@mgogoulos what's the ETA on those new docs?
@styiannis what's your quick response on how to change font colors?
I wound up looking up the green color in my browser and just find/replacing it with my chosen color in static/css/_commons.css.

**Reply by Subzero710** [reply]
index.scss   --default-theme-color: #009933; and --default-brand-color: #009933;
